aspnetcore/mvc/razor-pages/index.md

```diff
@@ -27,20 +25,48 @@ If you're looking for a tutorial that uses the Model-View-Controller approach, s
 
 [!INCLUDE[install 2.0](../../includes/install2.0.md)]
 
+<a name="rpvs17"></a>
+
+## Creating a Razor Pages project
+
+Select the **.NET Core CLI** tab if you are working with:
```

Comment:
Some context is missing here -- where is this tab?  And how does a tab make sense if you're using the CLI?